Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/ApplicantController.java

```diff
@@ -1,38 +1,29 @@
 package com.hydraulic.applyforme.controller;
 
-import com.hydraulic.applyforme.model.response.base.ApplyForMeResponse;
-import com.hydraulic.applyforme.service.ApplicantService;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.hydraulic.applyforme.model.domain.Member;
+import com.hydraulic.applyforme.service.impl.ApplicantServiceImpl;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import static com.hydraulic.applyforme.constants.PagingConstants.*;
-import static com.hydraulic.applyforme.constants.PagingConstants.DEFAULT_SORT_DIRECTION;
-
 @RestController
 @RequestMapping(
-        value = "applicant",
-        produces = { MediaType.APPLICATION_JSON_VALUE }
-)
+        value = "applicant/member",
+        produces = { MediaType.APPLICATION_JSON_VALUE })
 public class ApplicantController {
+    private final ApplicantServiceImpl applicantService;
 
-    private final ApplicantService service;
-
-    @Autowired
-    public ApplicantController(ApplicantService service) {
-        this.service = service;
+    public ApplicantController(ApplicantServiceImpl applicantService) {
+        this.applicantService = applicantService;
     }
 
-
-    @GetMapping("/entries")
-    public ApplyForMeResponse getAllSubmission(
-            @RequestParam(value = "pageNo", defaultValue = DEFAULT_PAGE_NUMBER, required = false) int pageNo,
-            @RequestParam(value = "pageSize", defaultValue = DEFAULT_PAGE_SIZE, required = false) int pageSize,
-            @RequestParam(value = "sortBy", defaultValue = DEFAULT_SORT_BY, required = false) String sortBy,
-            @RequestParam(value = "sortDir", defaultValue = DEFAULT_SORT_DIRECTION, required = false) String sortDir) {
```

Comment:
remove the id in the url, we do not need to specify the id of an existing user, instead we fetch the user from the security context.  so use the security context to get the id of the user and pass that into the service method

Reply:
I've implemented that and it works